src/ScottPlot5/ScottPlot5/Plottables/Heatmap.cs

```diff
@@ -258,7 +269,23 @@ public double GetValue(Coordinates coordinates)
 
     public IEnumerable<LegendItem> LegendItems => Enumerable.Empty<LegendItem>();
 
-    public Range GetRange() => Range.GetRange(Intensities);
+    public Range GetRange()
+    {
+        if (UseForcedRange)
+        {
+            return ForcedRange;
+        }
+        if (IntensityRange == default(Range))
+        {
+            // Needs update.  Get the range from intensities.
+            return Range.GetRange(Intensities); 
+        }
+        return IntensityRange;
+    }
+
+    public Range IntensityRange { get; private set; }
+    public Range ForcedRange { get; set; }
+    public bool UseForcedRange { get; set; }
```

Comment:
I don't know if I like having a `UseForcedRange` in addition to `ForcedRange`. A user who has logic to switch the range around on the same heatmap (this is common in image processing, it's essentially a form of contrast-adjustment) will have to maintain two values that do essentially the same thing. This is especially fiddly if they're loading saved presets, which does happen a lot if they're dealing with the same kind of images which is pretty common for scientific heatmaps. This is actually how the TSA uses their X-rays, they have a bunch of preset ranges that show specific contraband better on their screens.

You could imagine how a user might set `ForcedRange` but not `UseForcedRange`, which will have no effect. Or, they could set `UseForcedRange` but not `ForcedRange`, which I believe will result in a range of [0,0], which will give a useless heatmap (if it doesn't crash from divide by zero, I haven't checked). This may sound pedantic, but I think we can eliminate this possibility by using a `Range? ForcedRange`. If it's null, it's disabled, if it's non-null, it's enabled. It becomes impossible to enable it without setting a range, and impossible to set a range without enabling it.

Reply:
> but I think we can eliminate this possibility by using a `Range? ForcedRange`

I'll make this change now and push to this PR 👍